x-pack/plugins/ml/public/application/trained_models/models_management/expanded_row.tsx

```diff
@@ -47,48 +48,62 @@ const badgeFormatter = (items: string[]) => {
   );
 };
 
-const formatterDictionary: Record<string, (value: any) => JSX.Element | string | undefined> = {
-  tags: badgeFormatter,
-  roles: badgeFormatter,
-  create_time: timeFormatter,
-  timestamp: timeFormatter,
-};
+export function useListItemsFormatter() {
+  const bytesFormatter = useFieldFormatter(FIELD_FORMAT_IDS.BYTES);
+  const dateFormatter = useFieldFormatter(FIELD_FORMAT_IDS.DATE);
 
-export function formatToListItems(
-  items: Record<string, unknown> | object
-): EuiDescriptionListProps['listItems'] {
-  return Object.entries(items)
-    .filter(([, value]) => isDefined(value))
-    .map(([title, value]) => {
-      if (title in formatterDictionary) {
-        return {
-          title,
-          description: formatterDictionary[title](value),
-        };
-      }
-      return {
-        title,
-        description:
-          typeof value === 'object' ? (
-            <EuiCodeBlock
-              language="json"
-              fontSize="s"
-              paddingSize="s"
-              overflowHeight={300}
-              isCopyable={false}
-            >
-              {JSON.stringify(value, null, 2)}
-            </EuiCodeBlock>
-          ) : (
-            value.toString()
-          ),
-      };
-    });
+  const formatterDictionary: Record<string, (value: any) => JSX.Element | string | undefined> =
```

Comment:
i know it's not new code, but is it possible to easily remove this `any`?

Reply:
maybe it's possible, but I haven't figured out how to infer the type